examples/webgl_loader_texture_dds.html

```diff
@@ -83,6 +83,23 @@
 				map6.anisotropy = 4;
 				map6.colorSpace = THREE.SRGBColorSpace;
 
+				const map7 = loader.load( 'textures/compressed/disturb_dx10_bc6h_signed_nomip.dds' );
+				map6.anisotropy = 4;
+				map6.colorSpace = THREE.SRGBColorSpace;
+
+				const map8 = loader.load( 'textures/compressed/disturb_dx10_bc6h_signed_mip.dds' );
+				map6.anisotropy = 4;
+				map6.colorSpace = THREE.SRGBColorSpace;
+
+				const map9 = loader.load( 'textures/compressed/disturb_dx10_bc6h_unsigned_nomip.dds' );
+				map6.anisotropy = 4;
+				map6.colorSpace = THREE.SRGBColorSpace;
+
+				const map10 = loader.load( 'textures/compressed/disturb_dx10_bc6h_unsigned_mip.dds' );
+				map6.anisotropy = 4;
+				map6.colorSpace = THREE.SRGBColorSpace;
```

Comment:
My (macOS) computer doesn't support BPTC to test this, but I think the correct color space here is probably THREE.LinearSRGBColorSpace? Shouldn't affect rendering either way, but just to avoid any confusion!